pkg/coscheduling/coscheduling.go

```diff
@@ -137,6 +138,30 @@ func (cs *Coscheduling) PreFilter(ctx context.Context, state *framework.CycleSta
 	return framework.NewStatus(framework.Success, "")
 }
 
+// PostFilter is use for rejecting the group pods if a pod can not pass resource check when filter.
+func (cs *Coscheduling) PostFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod,
+	filteredNodeStatusMap framework.NodeToStatusMap) (*framework.PostFilterResult, *framework.Status) {
+	// check if failed due to preFilter failed, if not, filteredNodeStatusMap should not be nil
+	if filteredNodeStatusMap != nil {
+		return &framework.PostFilterResult{}, framework.NewStatus(framework.Success, "")
+	}
+
+	pgName, pg := cs.pgMgr.GetPodGroup(pod)
+	if pg == nil {
+		klog.V(4).Info("Pod does not belong to any group")
+		return &framework.PostFilterResult{}, framework.NewStatus(framework.Success, "")
```

Comment:
return `Unschedulable` b/c the logic is able to come down here is b/c the previous PostFitler plugins returned non-successful status, we should be consistent with those statuses.

Reply:
Sorry, have not understood very well.

Reply:
I mean:
- suppose it's the 2nd PostFilter plugin: why the logic comes down here is b/c the 1st PostFilter failed (in terms of defaultpreemption, it cannot find any candidate), so returning `Success` actually indicate it's a successful running, which isn't the case, is it?
- more generally, suppose it's configured as the Nth PostFilter plugins, returning `Success` will prevent further PostFilter plugins from their running, which doesn't quite make sense.

Reply:
thanks  will change this pr